src/pages/contact-us/index.js

```diff
@@ -0,0 +1,124 @@
+import React, { useReducer } from 'react'
+import { css } from 'styled-components'
+import { Button } from '../../components/elements'
+import Input from '../../components/Input'
+import Layout from '../../components/layout'
+import Subtitle from '../../components/Subtitle'
+import {
+  FlexDiv,
+  FlexDivWrap,
+  SidePanel,
+  H1Inverted,
+  HeadingSection,
+} from './styles'
+
+const CHANGE = 'CHANGE'
+
+const fields = [
+  { name: 'name', label: 'Full name' },
+  { name: 'organization', label: 'Company or organisation' },
+  { name: 'email', type: 'email', label: 'Email address' },
+  { name: 'phone-number', type: 'tel', label: 'Phone number' },
+]
+
+const reducer = (s, action) => {
```

Comment:
What is s?

Reply:
I know it's the state but come on it took me a minute to understand that

Reply:
if eslint is giving you errors name it in a different understandable way or name this state and change the other one to formData for example

Reply:
its a state in a reducer.. its naturally for state

Reply:
hmmm we know that but using one letter for your variable is not a sign for readable code, it is a suggestion after all